resources/web/style/base_styles.pcss

```diff
@@ -7,6 +7,10 @@
     margin: 1em 0;
   }
 
+  .blockquote {
+    font-size: 1rem;
```

Comment:
So, this is basically the size of the normal font in a regular docs page, right? It's 100% of the root font size, if I got this right.

Reply:
That's correct. For non-mobile devices, that will be 16px which was the previous size.

Reply:
Here's a preview if that's helpful: http://docs_1735.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/sql-client-apps-dbeaver.html

Reply:
Perfect.